services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -104,6 +104,7 @@ export const [useTag, updateTag] = createStore<TagsResponse>({ response: tagsRes
 
 const summaryResponse: GetProductSummaryResponse = { productSummary: [] };
 export const getSummary = (commitHash: string, environment: string, environmentGroup: string): void => {
+    updateSummary.set({ summaryReady: false });
```

Comment:
please add some instructions to the readme about setting up the manifest repo locally.
Something like "git tag foo-my-tag1 && git push --tags", so that we have it locally as well.